AirCasting/Dashboard/DashboardView.swift

```diff
@@ -15,7 +15,7 @@ struct DashboardView: View {
     @StateObject var coreDataHook: CoreDataHook
     @FetchRequest<SensorThreshold>(sortDescriptors: [.init(key: "sensorName", ascending: true)]) var thresholds
     // TODO: - We can rethink the way how "Select Section" works and change it
-    @EnvironmentObject var selectedSection_: SelectSection
+    @EnvironmentObject var selectedSectionNeededForSwipingImplementation: SelectSection
```

Comment:
Isn't this selected section the one that was used previously? I think this other selectedSection was added with this feature, wasn't it?

Reply:
Good catch Ania, thanks :)